src/python/visclaw/data.py

```diff
@@ -289,7 +291,7 @@ def getgauge(self, gaugeno, outdir=None, verbose=True):
                     print "continuing..."
                     return None
                     
-                gauge_data = amrclaw.GaugeData()
+                gauge_data = amrclaw.GaugeData(self.num_dim)
                 gauge_data.read(outdir)
 
```

Comment:
I think this is the way to go (see my "other" comment... I really need to learn to use the referencing and syntax highlighting here).